### PR TITLE
Set up axios to retrieve data from url endpoints

### PR DESCRIPTION
**Pull request by zxenonx**

### CODE REVIEW

File: frontend/src/components/MessageDisplaySection.vue [hunk 1]
```diff
@@ -1,23 +1,52 @@
...
+  },
+  mounted() {
+    // This will be called when MessageDisplaySection is loaded
+    this.getMessages(); // Call our getMessages function below
+  },
+  methods: {
+    getMessages() {
+      axios({
+        method: "get",
+        url: "http://127.0.0.1:8000/messages",
```
Comment (Delight-Fela-Steve) [inline frontend/src/components/MessageDisplaySection.vue:41]: shouldn't this be "http://dm.zuri.chat/messages"

File: zc_plugin_dm/settings.py [hunk 2]
```diff
@@ -34,6 +33,8 @@
 
 ALLOWED_HOSTS = ["dm.zuri.chat", '127.0.0.1']
 
+CORS_ORIGIN_ALLOW_ALL = True
```
Comment (Delight-Fela-Steve) [inline zc_plugin_dm/settings.py:36]: Are we safe?

File: frontend/src/components/MessageDisplaySection.vue (hunk 1 above)
Comment (zxenonx) [reply frontend/src/components/MessageDisplaySection.vue:41]: yeah, but currently we're working in a dev environment. In prod, we'll switch to dm subdomain.

File: zc_plugin_dm/settings.py (hunk 2 above)
Comment (zxenonx) [reply zc_plugin_dm/settings.py:36]: No, sorry. I'm going to fix it.

Comment (Delight-Fela-Steve) [reply zc_plugin_dm/settings.py:36]: kindly comment when done

Comment (zxenonx) [reply zc_plugin_dm/settings.py]: Done

**Comment by blacdev** [comment]
approved

**Pull request merged**